src/Billing/BillingReport.php

```
@@ -122,10 +122,13 @@ public static function generateTheQueryPart($daysheet = false)
     }
 
     //date must be in nice format (e.g. 2002-07-11)
+    /**
+     * @return mixed[]
+     */
     public static function getBillsBetween(
         $code_type,
         $cols = "id,date,pid,code_type,code,user,authorized,x12_partner_id"
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$cols'. (undefined)

(UnusedFormalParameter)